### PR TITLE
How to configure an API Connector

### LINKED ISSUE #2: building a connector - @thethan

### PR DESCRIPTION
**Pull request by thethan**
Basic example and description of creating a connector through an example of an api in mind. 

Resolves #2

### CODE REVIEW

File: connectors.mdx [hunk 1]
```diff
@@ -1,4 +1,80 @@
 ---
 title: 'Connectors'
-description: 'Learn how to create and manage your Canopies'
----
\ No newline at end of file
+description: 'Learn how to create and manage your Connectors'
+---
+
+<Info>
+    **Prerequisite** Keep in mind an api you want to hit and have documentation ready for that api.
```
Comment (alix-ryu-canopy) [inline connectors.mdx:7]: hit -> make requests to,

File: connectors.mdx [hunk 2]
```diff
@@ -1,4 +1,80 @@
...
+
+Repeat the steps to add more resources to the connector.
+
+<Frame>
+    <img src="/images/connector/configured-sidebar.png" style={{borderRadius: '0.5rem'}}/>
+</Frame>
+
+## Configure you connector
+
+From the sources tab. Add the connector to your project.
```
Comment (alix-ryu-canopy) [inline connectors.mdx:79]: Worth mentioning that to do this, you have to first select your project on the Projects sidebar.

**Review by alix-ryu-canopy — COMMENTED** [review]
Generally looks good! Made a few comments. Once merged, please share it with Mari so that she can read through, provide additional feedback, and make edits.

File: connectors.mdx (hunk 2 above)
Comment (thethan) [reply connectors.mdx:79]: Technically you do not. And kind of left this vague as it will probably be something we get rid of.

**Pull request merged**